docs/.vuepress/sidebar.js

```diff
@@ -33,13 +33,32 @@ module.exports = {
     'reference/'
   ],
   '/reference': [
-    'guide/introduction.md',
+    'guide/',
     {
       title: 'Reference',
       path: '/reference',
       collapsable: false,
       children: [
         'reference/glossary',
+        {
+          title: 'API routes',
+          path: '/reference/api',
+          children: [
+            'reference/api/authentication',
+            'reference/api/pieces',
+            'reference/api/pages',
+            'reference/api/media',
+            'reference/api/field-formats'
+          ]
+        },
+        {
+          title: 'Module API',
```

Comment:
It seemed like the general module API should be before the fields types (a subsection of modules).